manager/state/store/memory.go

```diff
@@ -76,8 +77,38 @@ var (
 	// WedgeTimeout is the maximum amount of time the store lock may be
 	// held before declaring a suspected deadlock.
 	WedgeTimeout = 30 * time.Second
+
+	// update()/write txn latency timer.
+	updateLatencyTimer metrics.Timer
```

Comment:
The variance on these values might be too high to be well covered by the `HistogramVec` default buckets. The defaults are in https://github.com/prometheus/client_golang/blob/master/prometheus/histogram.go#L59. The max bucket is 10 seconds, which should cover the large majority of use cases. If we need granularity over 10 seconds, this might be a problem.

Reply:
Agreed, but this is unrelated to this change and can be addressed separately.